server/src/main/resources/views/forbidden.html

```diff
@@ -21,8 +20,7 @@
                </div>
                <div class="col-md-10">
                     <h1>Forbidden</h1>
-                    <p>Please Login to continue browsing.</p>
-                    <a class="btn btn-danger" href="/oauth/login/google">Login</a>
```

Comment:
clean up and not part of this story. I removed the login button from Forbidden.html, since an unauth user cannot land on this page.